admin/assets/stylesheets/elements/_accessible-autocomplete.scss

```diff
@@ -0,0 +1,134 @@
+.autocomplete__wrapper {
```

Comment:
This is the css code that govuk accessible component uses which unfortunately is not by default offered as a scss module.